ls1intum/Artemis
# [Feature] [Orion] Assessment in Orion

### Checklist

### Motivation and Context
This change enables Orion to edit assessment comments locally. Its Artemis' half of https://github.com/ls1intum/Orion/pull/64.

### Description
Adds a new OrionTutorAssessmentComponent which will be displayed to an Orion user instead of the CodeEditorTutorAssessmentContainerComponent. This component does not show the code editor and does send and recieve relevant data from Orion.

### Steps for Testing
These are the testing steps to ensure no regression happened, for the feature testing please go to the Orion PR

1. Log in to Artemis
2. Create/Locate a programming exercise with several submissions
3. Assess the submissions
4. Make sure especially the assessment page with the code editor behaves as expected

### Review Progress
- Code Review
- Manual Tests

### Test Coverage
Everything in /orion: 100%

### Screenshots
Refer to the Orion PR
